content/reference/stacks.md

```diff
@@ -0,0 +1,195 @@
+
+SweetOps utilizes YAML configuration for stacks because it's an easy to work with format that is nicely portable across multiple tools. The stack yaml format is natively supported today via [atmos](https://github.com/cloudposse/atmos), [the terraform-yaml-stack-config module](https://github.com/cloudposse/terraform-yaml-stack-config), and [Spacelift](https://spacelift.io) via [the terraform-spacelift-cloud-infrastructure-automation module](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation).
+
+# Conventions
+
+We have a number of important conventions around stacks that are worth noting.
+
+## Stack Files
+
+Stack files can be very numerous in large cloud environments (think many dozens to hundreds of stack files). To enable proper organization of stack files, SweetOps recommends the following:
+
+* All stacks should be stored in a `stacks/` folder at the root of your infrastructure repository.
+* Global values you that you want to share across all stacks should go into a `stacks/globals.yaml` file
+  * This file name is special in that it will not be treated as a stack by any SweetOps automation tooling.
+* Name individual environment stacks as `$env-$stage.yaml`
+  * For example, `$env` could be `ue2` (for `us-east-2`) and `$stage` might be `prod` which would result in `stacks/ue2-prod.yaml`
+* When you have configuration that should be shared across `env`'s, you should use `gbl` as your `env` and then share that configuration using `import`.
+  * For example, you have values you want to share across `ue2-stage.yaml` and `uw2-stage.yaml` so you create a `glb-stage.yaml` stack file and utilize `import` in both `ue2` and `uw2` stacks to pull in that configuration.
+
+## Terraform Workspace Names
+
+`atmos` and accompanying terraform automation modules like [terraform-spacelift-cloud-infrastructure-automation](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation) will automatically create terraform [workspaces](https://www.terraform.io/docs/language/state/workspaces.html) when managing components. These workspaces derive their names from the stack name and the component name in question following this pattern: `$env-$stage-$component`. The result is workspace names like `ue2-dev-eks` or `uw2-prod-mq-broker`.
+
+<!--
+
+TODO: Do we want to define the folder structure of `stacks/` and subdirectories for catalogs here or elsewhere? I'm not fully spun up on that convention right now, so going to leave this as a todo item for discussion.
+
+## Folder Structure
+-->
+
+# Stack Schema
+
+<!--
+TODO: Is the vault backend supported as of today?
+TODO: Should we mention the `remote` backend considering the current state of the TFC automation module?
+TODO: Do we have a defined JSON schema document to share here for Stack validation?
```

Comment:
@dylanbannon if I'm not mistaken, you were working on this? Mind directing me to this json schema if it exists? I'd like provide it here for folks.

Reply:
https://github.com/cloudposse/atmos/blob/master/docs/schema/stack-config-schema.json

Reply:
Good stuff I will add this 👍